web/styles/popovers.css

```diff
@@ -1213,3 +1213,16 @@ ul.navbar-dropdown-menu-outer-list {
         }
     }
 }
+
+#gear-menu-dropdown {
+    .light-theme,
+    .dark-theme {
```

Comment:
This is not new in this commit, but these class names might be better as something more unique, like `gear-menu-select-light-theme`